ob/kustomize/overlay/defaults/kustomization.yaml

```diff
@@ -35,7 +35,7 @@ patches:
       metadata:
         name: openig-secrets-env
 resources:
-  - https://github.com/SecureApiGateway/sapig-openbanking-uk-developer-envs/core/kustomize/overlay/7.3.0/defaults?ref=master
+  - https://github.com/SecureApiGateway/sapig-openbanking-uk-developer-envs/core/kustomize/overlay/defaults?ref=1511-template-improvements
```

Comment:
I assume you need to update ref once other PR is merged?

Reply:
Oh wait, it's referencing this repo?

Reply:
Yes that needed to go back to being master, this is where the OB files uses Core as it's base and adds whatever is needed

Reply:
Ah yes, I assume you can't do a relative path then?

Reply:
You can, but using it like this to mirror relenvs release repo